test/verify/check-system-journal

```diff
@@ -643,8 +644,15 @@ ExecStart=/bin/sh -c 'sleep 5; for s in $(seq 10); do echo SLOW; sleep 0.1; done
         b.click("#abrt-reporting .pf-l-split:contains('Report to Cockpit') button:contains('Report')")
 
         test_user = 'correcthorsebatterystaple'
+        fields = ['text', 'password']
 
-        for purpose in ['text', 'password']:
+        # Since libreport version `2.15.2-7` bugzilla plugin does not require username and password but API key
+        # See https://github.com/abrt/libreport/commit/7c3c5d1ae1c39f0a3a3e9730bd25643a3f04fd50
+        libreport_version = m.execute('rpm -qa --qf "%{VERSION}-%{RELEASE}" libreport-plugin-bugzilla').rsplit(".", 1)[0]
+        if version.parse(libreport_version) > version.parse("2.15.2-6"):
```

Comment:
Could this be the new default, and we handle user+password in fedora-35 (and possibly fedora-36, didn't check) as the "special case"? Then it's easier to spot with grepping the tests for these OS names, and we can clean it up over time.

Reply:
I made a calendar entry in a month to clean this up.